clientSide/src/ui-components/CRColumn.jsx

```diff
@@ -0,0 +1,7 @@
+const CRColumn = (props) => {
+  const { className, children } = props;
+
+  return <div className={`cr-column ${className}`}>{children}</div>;
+};
+
+export default CRColumn;
```

Comment:
I like the idea of having Column and Container as components as it takes away a lot of boiler plate styling.
CR make no sense though.